src/styles/blocks/calendar.scss

```
@@ -0,0 +1,72 @@
+$dayWidth: 100px;
+$dayGap: 1px;
+
+body {
+  margin: 0;
+}
+
+.page__calendar {
+  margin: 0 auto;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding-top: 30px;
+  gap: 1px;
+  width: 7 * $dayWidth + 6 * $dayGap;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    width: $dayWidth;
+    height: $dayWidth;
+
+    box-sizing: border-box;
+    border: 1px solid black;
```

don't forget to create variables for all colors

```suggestion
    border: 1px solid $c-black;
```